addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -275,6 +284,7 @@ options.registry.gallery = options.Class.extend({
             src: img.getAttribute('src'),
             alt: img.getAttribute('alt'),
         }));
+        // TODO in master : replace 'data-interval' by 'data-bs-interval'.
```

Comment:
Why doing that part in master, is not it broken when using data-internal? :thinking: Worst case you could do "bsInterval or interval" but here it's option's js combined to the `website.gallery.slideshow` which is static and already using bs-interval anyway, I don't see why you cannot fix it directly :)